platforms/jvm/language-groovy/src/integTest/groovy/org/gradle/groovy/compile/GroovyCompileToolchainIntegrationTest.groovy

```diff
@@ -236,7 +236,7 @@ class GroovyCompileToolchainIntegrationTest extends MultiVersionIntegrationSpec
         JavaVersion.forClass(classFile("groovy", "test", "GroovySpec.class").bytes) == groovyTarget
 
         where:
-        javaVersion << JavaVersion.values().findAll { JavaVersion.VERSION_1_8 <= it && it <= JavaVersion.VERSION_23 }
+        javaVersion << JavaVersion.values().findAll { JavaVersion.VERSION_1_8 <= it && it <= JavaVersion.VERSION_24 }
```

Comment:
Can we move this to GroovyCoverage somehow?  That would keep updates in one place

Reply:
I'm not sure what you mean by this? There are no updates to GroovyCoverage right now. Are you saying that it would be nice to have all potential places that need updating in one spot?

Reply:
Yes because GroovyCoverage already has some knowledge about which versions of Java work with which versions of Groovy.  This seems like it's doing something similar.